docs/controls/WebView.md

```diff
@@ -309,6 +309,16 @@ Scripts in the web view content can use **window.external.notify** with a string
 
 You can use the [Settings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.settings) property (of type [WebViewControlSettings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrolsettings) to control whether JavaScript and IndexedDB are enabled. For example, if you use a web view to display strictly static content, you might want to disable JavaScript for best performance.
 
- 
+ ## Requirements
```

Comment:
@nmetulev Where should we put the information regarding the high DPI issues / workarounds (setting app.config values for WinForms, setting manifest information for WinForms and WPF)?

Reply:
Using a NOTE or IMPORTANT seems appropriate for this. Here is the syntax:

```
> [!NOTE]
Some note

> [!IMPORTANT]
Some important note

> [!WARNING]
Some warning note
```

For example, here how it renders in docs:
![image](https://user-images.githubusercontent.com/711864/39198908-1d81270c-479d-11e8-8160-694b89545723.png)

Reply:
@normesta I'll write something up and include it here for review

Reply:
@nmetulev @normesta Included in 30ad73c